app-misc/lirc/lirc-0.9.4a-r2.ebuild

```diff
@@ -91,6 +94,10 @@ src_install() {
 	if ! use static-libs; then
 		rm "${D}/usr/$(get_libdir)/liblirc_client.la" || die
```

Comment:
I might have asked this before (excuse my brain), have you checked whether LIRC installs `.pc` files? If so, nuke these libtool archives unconditionally, if not leave the conditional for the moment. If they really depend on `.la` files for recording the private dependencies, could you work with upstream to get them to use pkg-config instead? `.la` files are awful and need to be annihilated.